_protocols/new-dna-extraction.md

```diff
@@ -20,105 +20,113 @@ Needed consumables:
 10. Centrifuge 4000 rpm 10 min 4°C
 11. Discard supernatant
 12. Resuspend the cells (with ca. 1ml 1M TrisHCl buffer pH 7.4) 
-13. Transfer the solution into a 1.5 ml Eppendorf tube.
+13. Transfer the solution into a 1.5 ml Eppendorf tube (eppi).
    ![](/images/protocols/beer-dna-extraction/transfer_to_eppendorf.svg){: width="20%"}
-
-14. Weigth pellets (fresh empty 1.5ml eppi as tara): 6 eppis, weights between 30mg and 60mg 
-15. Pool pellets from two eppis to achieve ca. 70-90mg pellet per eppi (final # of eppis: 3)
+14. Centrifuge 4000 rpm 10 min 4°C
+15. Discard supernatant
+16. Weigh each of the 6 eppy pellets (use empty 1.5ml eppi as tara): weights of one pellet between 30mg and 60mg 
+17. Resolve the pellets by adding 500 μL and pipet up and down 
+18. Combine the solution of two eppis into one to achieve ca. 70-90mg pellet per eppi (final # of eppis: 3)
+19. Centrifuge 4000 rpm 10 min 4°C
+20. Discard supernatant
 
 
 
 ### STEP 2: Break-down the yeast cell wall – first round
-e now want to get the DNA out the yeast. The DNA is well protected by the membrane of the nucleus and the membrane of the cell. We need to break the membrane of the yeast and then the menbrane of the nucleus.
+Now, we want to get the DNA out the yeast. The DNA is well protected by the cell wall and the membrane of the nucleus. We need to break the membrane of the yeast and then the menbrane of the nucleus.
 
 ![](/images/protocols/beer-dna-extraction/yeast_cell.svg){: width="50%"}
 
 *A yeast cell - Frankie Robertson, CC ASA, [Wikimedia](https://en.wikipedia.org/wiki/File:Yeast_cell_english.svg)*
 
-Here: incubation times can be prolonged for difficult to lyse organisms
-- Suspend cells in an appropriate amount of the Y-PER Reagent. Scale the amount of Y-PER Reagent accordingly, maintaining a ratio of 8μL/1mg pellet. 
-	- We assumed all pellets correspond to 80 mg and added 640µl Y-PER
-- Mix by pipetting up and down until the mixture is homogenous
+- Suspend cells:
+ 1. add an appropriate amount of the Y-PER Reagent. Scale the amount of Y-PER Reagent accordingly, maintaining a ratio of 8μL/1mg pellet. 
+	- We assume all pellets correspond to 80 mg and added 640µl Y-PER
+ 2. Mix by pipetting up and down until the mixture is homogenous
 - Incubate at 65°C for 10 minutes. (can be extended – SHOULD BE EXTENDED??)
+- Centrifuge at 13,000 × g for 5 minutes 
+- Discard supernatant
 
 
 ### STEP 3: Break-down the yeast cell wall – second round
-Here: incubation times can be prolonged for difficult to lyse organisms
-- Centrifuge at 13,000 × g for 5 minutes, 
-- Discard supernatant,
-- Add 400μL of DNA Releasing Reagent A, 
+
+- Add 400μL of DNA Releasing Reagent A
 - Add 400μL of DNA Releasing Reagent B
 - Mix by pipetting up and down until the mixture is homogenous
 - Incubate at 65°C for 10 minutes. (can be extended – SHOULD BE EXTENDED??)
 
 
 ### STEP 4: Stop protein activity in the solution
 - Add 200μL of Protein Removal Reagent to mixture
```

Comment:
Protein Removal Reagent is probably a protease? We usually add protease to digest proteins before DNA precipitation. Degraded proteins are insoluble and can therefore be pelleted.